### ISSUE TITLE
BUG: Add authentication for login 

### ISSUE THREAD

**Issue by MitulSonagara** [Anushkabh/krishiconnect#387]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Currently if i login as seller. Then even if i logout and press back then i can go to seller dashboard. There should be constraints so that no one can directly access the home page.
Currently id i direct search for this url https://anushkabh.github.io/krishiconnect/home.html it goes to home page for seller which must not happen without signup and login.
So basically there are issues in backend.

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by MitulSonagara** [comment]
@RitiChandak for this I have to connect backend to this project and also add database for this.

**Comment by RitiChandak** [comment]
@Anushkabh Please take the decision

**Comment by MitulSonagara** [comment]
@Anushkabh and @RitiChandak Yes, Please take the decision, I am sure enough that i can to this job easily. I have created many full stack projects. I will make this project with fully integrated with backend and make PRs step by step.
